libiocage/lib/JailConfig.py

```diff
@@ -233,6 +233,37 @@ def _get_priority(self) -> int:
     def _set_priority(self, value: typing.Union[int, str]):
         self.data["priority"] = str(value)
 
+    # legacy support
+    def _get_tag(self) -> str:
+
+        if self._has_legacy_tag is True:
+            return self.data["tag"]
+
+        try:
+            return self["tags"][0]
+        except KeyError:
+            return None
```

Comment:
how is this even possible?

Reply:
when not tag was set at all, tags is empty.. an if clause would have been the better fit

Reply:
what i mean is, we declare `-> str`, but our return type is `Optional[str]`

Reply:
Got you! This must be the case in other places as well. https://github.com/iocage/libiocage/issues/86 is the ticket for it.